docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,234 @@
+4. Once the folder is imported, click on the big **REPO** tab button next to *"Resource Overview"*.
+5. Then, click *"Load REPO from RPKGS folder"*. Now you’ll see everything neatly ordered with a good overview of each category. This is the **Repository**.
+6. Expand the *Outfits* section below. Search for your outfit there. Or use the search bar on top.
+
+>You can leave your **RPKG** program open after finding your suit
+
+### Method 2: Using GlacierKit
+
+1. Open **GlacierKit**
+2. Select the *"game contents"* tab on the left.
+3. - Copy the following **repository's** file hash  ```00204D1AFD76AB13``` and paste it on the search bar and hit enter. 
+   - Or simply type ***pro.repo*** on the search bar and hit enter.
+Both methods should bring you to a single file called **pro.repo**. 
+4. Open it by clicking on it (obviously) and then click on *"Open in editor"* in the new menu that popped up on the right. This is the ***repository*** but in **Glacierkit**
+
+:::tip
+You now know the exact name of the **repository** file. Next time you want to open the **repository** in **GlacierKit** just type ***pro.repo***.
+::: 
+
+5. You can then start searching for your suit below in the *"Unmodified section"*.
+6. Double check if you have the right suit selected by looking up the image in **GlacierKit**.
+To do so, simply 
+
+   - find the correct image code line on the right that starts with ***"Image":".....***
+
+ 
+
+   - and copy the ***assembly path*** 
+
+   The **assembly path** is the line after **"image":"...**" inside the quotation.
+
+     Example of a full image code: 
+   ```json
+   "Image": "images/unlockables_override/47_outfits_bloodmoney_gloves.jpg"
+   ```
+   and its **assembly path**
+   ```json
+   images/unlockables_override/47_outfits_bloodmoney_gloves.jpg
+   ```
+
+- Paste that in the search bar in the *game contents* tab on the left 
+- and hit enter on your keyboard.
+
+
+>If the image you found matches the suit you want to modify, you're all set for the next step and you can go back to your **repository** file. (Where you saw your code).
+
+>You can leave open your **GlacierKit** program
+
+:::info GlacierKit vs. RPKG
+While **RPKG** is helpful for viewing images in the **repository**, it’s best to work primarily in **GlacierKit**, as it will be your main modding tool. The **repository code lines** in both tools are identical, so practicing in **GlacierKit** is encouraged for efficient modding.
+:::
+
+
+---
+## STEP 2: **FIND THE IMAGE**
+In this step we will have to look for the correct Image code and remember its ***assembly path***.
+:::note
+You already did this step in the **GlacierKit** method but follow along
+:::
+   ### 1. Open your file
+Click on your suit and you should see its **repository** code on the right
+
+   ### 2. Locate the assembly path
+There will be a line of code starting with `"Image":"....`
+
+   :::example
```

Comment:
```
Warning:  Docusaurus found 1 unused Markdown directives in file "docs/modding/suitmodding/addingblobs.md"
- :::example (106:4)
```

I think maybe having it tabbed is causing this error. Not really too sure though.

Reply:
I will just remove it since it's literally part of the tutorial. I thought example admonition was real